packages/blade/src/utils/dataAttribute/index.native.ts

```diff
@@ -0,0 +1 @@
+export * from './dataAttribute.native';
```

Comment:
why 2 index files? is that how we write in module suffixes world?

Reply:
Yes, 1 for web, 1 for native. Notice how we don't need another `dataAttribute.d.ts` file and then reexport the function again from there, much simpler & less confusing. 

You can check out the "Module Resolution Example" section at the end of [TypeScript RFC](https://github.com/razorpay/blade/blob/master/rfcs/writing-cross-platform-typescript.md)

<img width="789" alt="image" src="https://user-images.githubusercontent.com/35374649/203251761-8ba41e2a-e7d6-4a8a-bed0-51d2b1bc37c3.png">